packages/@atjson/document/src/annotation.ts

```diff
@@ -117,8 +130,10 @@ export default abstract class Annotation<Attributes = {}> {
     let AnnotationClass = this.getAnnotationConstructor();
     let AnnotationToCompareClass = annotationToCompare.getAnnotationConstructor();
 
-    let lhsAnnotationAttributes = this.attributes;
-    let rhsAnnotationAttributes = annotationToCompare.attributes;
+    let lhsAnnotationAttributes =
+      removeUndefinedValuesFromObject(this.attributes) || {};
```

Comment:
Attributes should be definitely defined; I would like to blow up loudly if `this.attributes` is `null` or `undefined`.

Reply:
roger 👍